### PR TITLE
sci-libs/vtk: fix build with gcc-13

### PR DESCRIPTION
**Pull request by waebbl**
add upstream patch with gcc-13 related fixes

Closes: https://bugs.gentoo.org/905387

### CODE REVIEW

File: sci-libs/vtk/files/vtk-9.2.5-More-include-cstdint-to-compile-with-gcc13.patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
+From 890f7574f01cf58cb1522772a62ca07deb438479 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline sci-libs/vtk/files/vtk-9.2.5-More-include-cstdint-to-compile-with-gcc13.patch:1]: No commit and bug links?

Comment (waebbl) [reply sci-libs/vtk/files/vtk-9.2.5-More-include-cstdint-to-compile-with-gcc13.patch:1]: Sorry, forgot to update this.

**Comment by thesamesam** [comment]
Fixed, thank you!

**Comment by waebbl** [comment]
> Fixed, thank you!

Thanks for adding the line. Was just rebasing to add it this morning when I suddenly got a merge conflict.

**Pull request closed without merging**